java/arcs/core/data/proto/annotation.proto

```diff
@@ -0,0 +1,17 @@
+syntax = "proto3";
+
+package arcs;
+
+option java_package = "arcs.core.data.proto";
+option java_multiple_files = true;
+
+// Represents the usage of an annotation.
+// TODO(b/155796088): Add support for all annotation functionality, including
+// annotation definitions and all parameter types.
+message AnnotationRefProto {
+  string name = 1;
+
+  // Maps from param name to value.
+  // TODO(b/155796088): Support parameter types other than just string.
```

Comment:
fyi you could already have number and boolean. or do you mean complex types, like objects?

Reply:
I meant more complex types. But you're right, I will add the other primitives in a follow up.